src/sentry/static/sentry/app/views/projectDetail/projectLatestReleases.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import {Location} from 'history';
+import pick from 'lodash/pick';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import {analytics} from 'app/utils/analytics';
+import {Organization, Release} from 'app/types';
+import {SectionHeading} from 'app/components/charts/styles';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import styled from 'app/styled';
+import {URL_PARAM} from 'app/constants/globalSelectionHeader';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import Placeholder from 'app/components/placeholder';
+import ButtonBar from 'app/components/buttonBar';
+import Button from 'app/components/button';
+import DateTime from 'app/components/dateTime';
+import Version from 'app/components/version';
+import {RELEASES_TOUR_STEPS} from 'app/views/releases/list/releaseLanding';
+import FeatureTourModal from 'app/components/modals/featureTourModal';
+
+const DOCS_URL = 'https://docs.sentry.io/product/releases/';
+
+type Props = AsyncComponent['props'] & {
+  organization: Organization;
+  projectSlug: string;
+  projectId: string | undefined;
+  location: Location;
+};
+
+type State = {
+  releases: Release[] | null;
+  hasOlderReleases: boolean;
+} & AsyncComponent['state'];
+
+class ProjectLatestReleases extends AsyncComponent<Props, State> {
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {location, organization, projectSlug} = this.props;
+
+    const query = {
+      ...pick(location.query, [...Object.values(URL_PARAM)]),
+      per_page: 5,
+    };
+
+    // TODO(project-detail): this does not filter releases for the given time
```

Comment:
Are the start/end/statsPeriod parameters not working here?

Reply:
No, maybe it's got something to do with the fact that start/end/statsPeriod on the releases page is used to recalculate metrics, not filter them.